src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+namespace Tizen.NUI
+{
+    using System;
+    using System.ComponentModel;
+    using System.Runtime.InteropServices;
+    using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
+    using Tizen.NUI.BaseComponents;
+
```

Comment:
Please add documents and editbrowsable tag in every public classes, properies, method, and enums.
check other cs files for the details.

        /// <summary>
        /// summary of class, property, method. 
        /// </summary>
        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
        [EditorBrowsable(EditorBrowsableState.Never)]
        
you can add reference by <see cref="reference class" />.

Reply:
커맨트 추가하였습니다.